modules/UI/recording/Recording.js

```diff
@@ -391,15 +411,15 @@ var Recording = {
 
             this._updateStatusLabel(this.recordingPendingKey, true);
         }
-        else if (recordingState === Status.ERROR) {
+        else if (recordingState === Status.ERROR
+                    || recordingState === Status.FAILED) {
             buttonSelector.removeClass(this.baseClass + " active");
             buttonSelector.addClass(this.baseClass);
 
             this._updateStatusLabel(this.recordingErrorKey, true);
+            console.log("Recording failed for the following reason: ", error);
```

Comment:
If we are using the error only for printing it in the console isn't it better to print it as soon as it is received from the library than passing it to methods as an argument?